sqlparse/sql.py

```diff
@@ -251,7 +216,7 @@ def _token_matching(self, funcs, start=0, end=None, reverse=False):
                 if func(token):
                     return token
 
-    def token_first(self, ignore_whitespace=True, ignore_comments=False):
+    def token_first(self, skip_ws=True, skip_cm=False):
```

Comment:
Why did you change the name of the keywords? I don't see any benefit (except that they are shorter). 

If there are any benefits in changing those keywords we also need to change the docstring and mention this change in the release notes in case third-party apps are using it.